app/jobs/calendar_importer/parsers/out_savvy.rb

```diff
@@ -5,46 +5,13 @@
 # Parent parser classes should not be added.
 
 module CalendarImporter::Parsers
-  class OutSavvy < Base
-    NAME = 'OutSavvy'
+  class OutSavvy < LdJson
```

Comment:
importers should have links back to the source event - maybe for another PR tho

Reply:
if you mean events having a supplied URL then that already happens.

But i think you mean a new field on events that has a "this was downloaded from this page" url (?) which is defo a new feature.

Reply:
the `publisher_url`?

Reply:
look at the comment further down this pr - there's a `footer` method